quic/s2n-quic/src/tests/recorder.rs

```diff
@@ -129,6 +129,34 @@ event_recorder!(
     }
 );
 
+event_recorder!(
+    InitialCryptoFrameReceived,
```

Comment:
Not sure I'm getting why you need to look at Crypto frames. Could you just take the `ConnectionStarted` event and take the `remote_cid` from that?